### ISSUE TITLE
try to systematically update API data

### ISSUE THREAD

**Issue by andrewsu** [biothings/biothings.api#235]
Many APIs haven't been updated since first creation. We should try to systematically go through our existing APIs and see where updates are available and easily applied (i.e., no data format changes). Ideally, we can have this done before our soon-to-be-submitted grant proposal is reviewed in ~September.

**Comment by colleenXu** [comment]
Related to this? https://github.com/biothings/pending.api/issues/25#issue-911909948 and the "On updating what we already have" section of this [comment](https://github.com/biothings/BioThings_Explorer_TRAPI/issues/372#issuecomment-1036582193) (this list is incomplete and outdated).

**Comment by colleenXu** [comment]
@everaldorodrigo @andrewsu @newgene 

Issue for Everaldo to consider working on

**Comment by newgene** [comment]
@everaldorodrigo this issue in general means we should set auto-build schedule for those pending.api data sources which are regularly updated and have relatively stable parsers. This does not means auto release, which we still want to trigger manually.

**Comment by everaldorodrigo** [comment]
The following data sources were configured to have auto-build:
- DISEASES
- TISSUES
- cell_ontology
- chebi
- clinicaltrials_gov
- dois
- ebi_gene2phenotype
- fda_drugs
- gwascatalog
- hpo
- ncit
- phewas
- uberon
- ufeno_ontology

**Comment by ctrl-schaff** [comment]
@everaldorodrigo do you think we can close this issue as we've migrated most of the issue handling to the pending.api repository?

**Comment by everaldorodrigo** [comment]
Yes, from my side we can close it. I think we just need to keep an eye on those datasources regularly and make sure they’re released as needed.

**Comment by ctrl-schaff** [comment]
Closing issue and migrating any further maintenance to the [pending.api repository](https://github.com/biothings/pending.api/issues)
